csi.proto

```diff
@@ -502,6 +510,34 @@ message ControllerServiceCapability {
     RPC rpc = 1;
   }
 }
+
+message ControllerExpandVolumeRequest {
+  // The ID of the volume to expand. This field is REQUIRED
+  string volume_id = 1;
+
+  // the current capacity of the volume in bytes.
+  // This field is OPTIONAL
+  int64 current_capacity_bytes = 2;
```

Comment:
What is the purpose of having the caller give the current capacity?

Reply:
Ditto, I don't understand why this is needed

Reply:
Because in many cases (such as glusterfs) - the volume expansion api call requires difference(new_size-current). It is possible that the SP can figure out the current size on its own but I don't know if that will be always possible.